include/triton-shared/Conversion/TritonArithToLinalg/ConversionPatterns.hpp

```diff
@@ -860,6 +860,44 @@ struct BitcastConverter : public OpConversionPattern<triton::BitcastOp> {
   }
 };
 
+struct CallConverter : public OpConversionPattern<triton::CallOp> {
+  using OpConversionPattern<triton::CallOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(triton::CallOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    SmallVector<Value> args = adaptor.getOperands();
+
+    // We need to bypass extra arguments added by addProgramInfo which are num_programs and program_ids
+    if (FuncOp parentFunc = op->getParentOfType<FuncOp>()) {
+      SymbolRefAttr calleeAttr = op.getCalleeAttr();
```

Comment:
nit: i know this file has `using namespace triton`, but would you mind prefixing triton::FuncOp to make it clear that we're referring to the triton func op? thank you

Reply:
I agree, it is better since Functions may belong to different namespaces, fixed, thank you!